R/calcIEA_WEO_2021.R

```diff
@@ -30,7 +26,7 @@ calcIEA_WEO_2021 <- function(aggregate, isValidation = FALSE) { # nolint
 
   mapping$variable <- trimws(mapping$variable)
 
-  data <- readSource("IEA_WEO_2021", subtype = aggregate)
+  data <- readSource("IEA_WEO_2021", subtype = subtype)
```

Comment:
`readIEA_WEO_2021()` does not take a `subtype` argument.

Reply:
But convertIEA_WEO_2021 does.

Reply:
Ah, ok.  Getting passed on to `convertIEA_WEO_2021()`.